### PR TITLE
add 1208

### PR DESCRIPTION
**Pull request by Woodii1998**

### CODE REVIEW

File: packages/engine/errors/1208.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+original: "'{0}' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module."
+excerpt: "You have set the 'isolatedModules' flag. In the 'A' file you write certain code that can’t be correctly interpreted by a single-file transpilation process. Add an import, export, or an empty 'export {}' statement to make it a module."
```
Comment (mattpocock) [inline packages/engine/errors/1208.md:3]: > In the 'A' file you write certain code that can’t be correctly interpreted by a single-file transpilation process

Could you explain this in a clearer way? Perhaps using less complicated language (interpreted, transiplation)

**Comment by mattpocock** [comment]
Thanks for the contribution!

File: packages/engine/errors/1208.md (hunk 1 above)
Comment (Woodii1998) [reply packages/engine/errors/1208.md:3]: Okay, I'll think about how to explain this

**Review by mattpocock — APPROVED** [review]

**Comment by mattpocock** [comment]
Brilliant, thanks!

**Pull request merged**